README.md

```diff
@@ -192,17 +192,22 @@ and reports back results to the queue.
                                             for serving live logs; see
                                             https://github.com/taskcluster/livelog and
                                             https://github.com/taskcluster/stateless-dns-server
-          rootURL                           The root URL of the Taskcluster deploment to which
+          rootURL                           The root URL of the taskcluster deployment to which
                                             clientId and accessToken grant access. For example,
-                                            'https://taskcluster.net'.
+                                            'https://taskcluster.net'. Individual services can
+                                            override this settings - see the *BaseURL settings.
           workerId                          A name to uniquely identify your worker.
           workerType                        This should match a worker_type managed by the
                                             provisioner you have specified.
 
         ** OPTIONAL ** properties
         =========================
 
-          authBaseURL                       The base URL for API calls to the auth service.
+          authBaseURL                       The base URL for taskcluster auth API calls.
+                                            If not provided, the base URL for API calls is
+                                            instead derived from rootURL setting as follows:
+                                              * https://auth.taskcluster.net/v1 for rootURL https://taskcluster.net
+                                              * <rootURL>/api/auth/v1 for all other rootURLs
```

Comment:
It seems like you could just leave this at "derived from the rootURL setting".  Within six months we'll be removing the taskcluster.net special-case.

Reply:
I think having the extra clarity is useful, and we can still remove it in 6 months if the special casing goes away.

Reply:
It's not an "if" :)